src/test/java/testresults/annual_certificates/TestPostTestResultsAnnualCertificateGenerationHgv.java

```diff
@@ -23,7 +23,6 @@
     @TestData
     public static Collection<Object[]> testData(){
         return Arrays.asList(new Object[][]{
-                {"warmup test", "warmup test", "95", 1, "pass", "ffv2"},
```

Comment:
Is this warm-up not needed?

Reply:
not at the moment, the test passes without it